### PR TITLE
Revisit Identify layer features sample

### PR DESCRIPTION
**Pull request by ruiqima**
## Description

PR to revisit sample to reflect new changes in Kotlin toolkit.

## Links and Data

#3582

## What To Review

- [ ] Build and run the sample to check for any errors.
- [ ] Update the MapViewModel to be able to work with the geo-compose module.
- [ ] Add the line to the "Additional information section": This sample uses the GeoCompose Toolkit module to be able to implement a Composable MapView.
- [ ] Replace the application parameter from the MainScreen with the use of LocalContext.current.applicationContext 
- [ ] Add the tags: geocompose, tookit
- [ ] Update the sample's metadata.json file to reflect file and tags changes

## How to Test

Run the sample on the sample viewer or the repo.

### CODE REVIEW

File: identify-layer-features/src/main/java/com/esri/arcgismaps/sample/identifylayerfeatures/components/MapViewModel.kt [hunk 1]
```diff
@@ -71,14 +74,25 @@ class MapViewModel(
...
+        map.apply {
+            // set initial Viewpoint to North America
+            initialViewpoint = Viewpoint(
+                center = Point(
+                    x = -10977012.785807,
+                    y = 4514257.550369,
+                    spatialReference = SpatialReference(wkid = 3857)
+                ),
+                scale = 68015210.0
+            )
```
Comment (shubham7109) [inline identify-layer-features/src/main/java/com/esri/arcgismaps/sample/identifylayerfeatures/components/MapViewModel.kt:86]: This viewpoint feels way tooo precise than it needs to be for this sample. Can we use the simple North America viewpoint as the other [samples](https://github.com/Esri/arcgis-maps-sdk-kotlin-samples/blob/4c3001260c2c047ec5b9496bd7a47dc732a6015f/manage-operational-layers/src/main/java/com/esri/arcgismaps/sample/manageoperationallayers/components/MapViewModel.kt#L66):
```suggestion
            // set initial Viewpoint to North America
            initialViewpoint = Viewpoint(39.8, -98.6, 5e7)
```

File: identify-layer-features/src/main/java/com/esri/arcgismaps/sample/identifylayerfeatures/components/MapViewModel.kt [hunk 2]
```diff
@@ -21,8 +21,11 @@ import androidx.compose.runtime.mutableStateOf
 import androidx.compose.ui.unit.dp
 import androidx.lifecycle.AndroidViewModel
 import com.arcgismaps.data.ServiceFeatureTable
+import com.arcgismaps.geometry.Point
+import com.arcgismaps.geometry.SpatialReference
```
Comment (shubham7109) [inline identify-layer-features/src/main/java/com/esri/arcgismaps/sample/identifylayerfeatures/components/MapViewModel.kt:25]: Unused imports 

```suggestion
```

**Review by shubham7109 — APPROVED** [review]
Changes look good to approve, two minor changes to make 👍

**Review by hud10837 — APPROVED** [review]

File: identify-layer-features/src/main/java/com/esri/arcgismaps/sample/identifylayerfeatures/components/MapViewModel.kt (hunk 2 above)
Comment (ruiqima) [reply identify-layer-features/src/main/java/com/esri/arcgismaps/sample/identifylayerfeatures/components/MapViewModel.kt:25]: Nice catch!

**Pull request merged**